microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/FaultToleranceMpService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.microprofile.faulttolerance;
+
+import io.helidon.microprofile.config.MpConfig;
+import io.helidon.microprofile.server.spi.MpService;
+import io.helidon.microprofile.server.spi.MpServiceContext;
+
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+
+/**
+ * Class FaultToleranceMpService.
+ */
+public class FaultToleranceMpService implements MpService {
+
+    @Override
+    public void configure(MpServiceContext mpServiceContext) {
+        MpConfig config = (MpConfig) ConfigProviderResolver.instance().getConfig();
+        CommandScheduler.create(config.getConfig().get("helidon.fault-tolerance.async-thread-pool"));
```

Comment:
Replace the lines with:
`        CommandScheduler.create(mpServiceContext.getHelidonConfig().get("helidon.fault-tolerance.async-thread-pool"));`

Reply:
Done